Tensile/Source/client/main.cpp

```diff
@@ -155,9 +156,11 @@ namespace Tensile
                 ("platform-idx",             po::value<int>()->default_value(0), "OpenCL Platform Index")
 
                 ("num-warmups",              po::value<int>()->default_value(0), "Number of warmups to run")
+                ("sync-after-warmups",       po::value<bool>()->default_value(true), "Synchronize GPU after warmup kernel runs")
                 ("num-benchmarks",           po::value<int>()->default_value(1), "Number of benchmarks to run")
                 ("num-enqueues-per-sync",    po::value<int>()->default_value(1), "Enqueues per sync")
```

Comment:
the comment for "num-enqueues-per-sync" should be revised as well since new argument "min-flops-per-sync" may change the behavior.